airbyte-webapp/src/packages/cloud/views/auth/Auth.module.scss

```
@@ -2,21 +2,22 @@
 
 .container {
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
   width: 100%;
+  height: 100%;
   background: colors.$white;
 }
 
 .leftSide {
+  display: flex;
+  flex-direction: column;
+  overflow-y: auto;
   width: 100%;
   padding: 20px 36px 39px 46px;
 }
 
 .rightSide {
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  width: 100%;
+  display: none;
```

:information_source: By default now hide the right side on small screens.